.travis.yml

```diff
@@ -36,6 +36,9 @@ matrix:
   - go: "1.12.1"
     env:
     - HOTROD=true
+  - go: "1.12.1"
+    env:
+    - TOKEN_INTEGRATION_TEST=true
```

Comment:
Would prefer it to be called `TOKEN_PROPAGATION_TEST`

Reply:
I'll integrate into ES tests as @pavolloffay suggested.